src/map/map.ts

```diff
@@ -102,6 +105,50 @@ const DEFAULT_CONFIG = {
         'hoverClosest3d',
         'tableRotation',
         'resetCameraLastSave3d',
+        'toImage',
+    ],
+
+    modeBarButtonsToAdd: [
+        {
+            name: 'Download as PNG',
+            icon: {
+                width: 400,
+                height: 424,
+                path: extractSvgPath(PNG_SVG),
+            },
+            click: function (gd: PlotlyScatterElement) {
+                Plotly.downloadImage(gd, {
+                    filename: 'newplot',
```

Comment:
How about something like this instead, to make the filename clearer?

```suggestion
                    filename: 'chemiscope-map',
```